website/content/en/docs/ansible/development-tips.md

```diff
@@ -323,44 +331,69 @@ $ kubectl create -f deploy/operator.yaml
 
 Verify that the foo-operator is up and running:
 
-```sh
+```none
 $ kubectl get deployment
-NAME                     DESIRED   CURRENT   UP-TO-DATE   AVAILABLE   AGE
+NAME               DESIRED   CURRENT   UP-TO-DATE   AVAILABLE   AGE
 foo-operator       1         1         1            1           1m
 ```
 
+Now you can create a Custom Resource instance of Foo with variable `state`
+default to `present`:
+
+```bash
+$ kubectl create -f deploy/crds/foo.example.com_v1alpha1_foo_cr.yaml
+```
+
+Check that ConfigMap `example-config` was created:
+
+```none
+$ kubectl get configmaps
+NAME                    STATUS    AGE
+example-config          Active    3s
+```
+
+<!-- TODO(Qiming): consolidate this subsection with quickstart -->
 #### Viewing the Ansible logs
 
-In order to see the logs from a particular you can run:
+In order to see the logs from a particular operator you can run:
 
-```sh
+```bash
 kubectl logs deployment/foo-operator
 ```
 
-The logs contain the information about the Ansible run and will make it much easier to debug issues within your Ansible tasks.
-Note that the logs will contain much more detailed information about the Ansible Operator's internals and interface with Kubernetes as well.
+The logs contain the information about the Ansible run and are useful for
+debugging your Ansible tasks. Note that the logs may contain much more
+detailed information about the Ansible Operator's internals and its
+interactions with Kubernetes as well.
 
-Also, you can use the environment variable `ANSIBLE_DEBUG_LOGS` set as `True` to check the full Ansible result in the logs in order to be able to debug it.
+Also, you can set the environment variable `ANSIBLE_DEBUG_LOGS` to `True` to
+check the full Ansible result in the logs in order to be able to debug it.
 
 **Example**
 
 In the `deploy/operator.yaml`:
+
 ```yaml
-...
-- name: ANSIBLE_DEBUG_LOGS
-  value: "True"
-...
+spec:
+  template:
+    spec:
+      containers:
+        - name: foo-operator
+          env:
```

Comment:
```suggestion
          ...
          env:
```

Just to make clear that has more code between the name and the env.

Reply:
That is doable, but I have already marked this code snippet as YAML. Adding "..." may break the syntax, I'm afraid.
This snippet is obviously not a complete one...

Anyway, if you insist, I can send another commit.

Reply:
It is the default standard that we work with to illustrate has more code and do not lead misunderstandings. you can check the other docs. Could you please just commit this suggestion? Then, for me it would be ok but before merge requires more one approval which means a second reviewer too.

Reply:
see 09c91b7